pkg/asset/ignition/bootstrap/bootstrap.go

```diff
@@ -265,6 +266,7 @@ func (a *Bootstrap) getTemplateData(installConfig *types.InstallConfig, releaseI
 		Registries:            registries,
 		BootImage:             string(*rhcosImage),
 		PlatformData:          platformData,
+		SingleNode:            *installConfig.ControlPlane.Replicas == 1,
```

Comment:
So regardless of single-node deployment type all of the changes gated by `SingleNode` are true? CRC, libvirt, etc.

Reply:
Unsure I understand the question.
In case SingleNode == true, bootkube.service.template will render a different bootkube.service.
This bootkube service will be packed into the bootstrap ignition.
When the user ignite an RHCoS image with this the bootkube service will execute the bootstrap logic and will "pivot" the node to become a master node and cleanup the bootstrap garbage, so we end up with a Single Node Openshift.

Reply:
I just mean this all works with other single node usages such as CRC right? The base condition for the logic being used is master replica 1. I was just pointing out it could affect others.